docs/getting-started-guides/kubeadm.md

```diff
@@ -121,104 +138,126 @@ kubeadm to tell it what to do.
-[tokens] Generated token: "064158.548b9ddb1d3fad3e"
-[certificates] Generated Certificate Authority key and certificate.
-[certificates] Generated API Server key and certificate
-[certificates] Generated Service Account signing keys
-[certificates] Created keys and certificates in "/etc/kubernetes/pki"
-[kubeconfig] Wrote KubeConfig file to disk: "/etc/kubernetes/kubelet.conf"
+[preflight] Starting the kubelet service
+[certificates] Generated CA certificate and key.
+[certificates] Generated API server certificate and key.
+[certificates] API Server serving cert is signed for DNS names [kubeadm-master kubernetes kubernetes.default kubernetes.default.svc kubernetes.default.svc.cluster.local] and IPs [10.96.0.1 10.138.0.4]
+[certificates] Generated API server kubelet client certificate and key.
+[certificates] Generated service account token signing key and public key.
+[certificates] Generated front-proxy CA certificate and key.
+[certificates] Generated front-proxy client certificate and key.
+[certificates] Valid certificates and keys now exist in "/etc/kubernetes/pki"
 [kubeconfig] Wrote KubeConfig file to disk: "/etc/kubernetes/admin.conf"
+[kubeconfig] Wrote KubeConfig file to disk: "/etc/kubernetes/kubelet.conf"
+[kubeconfig] Wrote KubeConfig file to disk: "/etc/kubernetes/controller-manager.conf"
+[kubeconfig] Wrote KubeConfig file to disk: "/etc/kubernetes/scheduler.conf"
 [apiclient] Created API client, waiting for the control plane to become ready
-[apiclient] All control plane components are healthy after 61.317580 seconds
+[apiclient] All control plane components are healthy after 16.772251 seconds
 [apiclient] Waiting for at least one node to register and become ready
-[apiclient] First node is ready after 6.556101 seconds
-[apiclient] Creating a test deployment
+[apiclient] First node is ready after 5.002536 seconds
 [apiclient] Test deployment succeeded
-[token-discovery] Created the kube-discovery deployment, waiting for it to become ready
-[token-discovery] kube-discovery is ready after 6.020980 seconds
+[token] Using token: <token>
+[apiconfig] Created RBAC rules
 [addons] Created essential addon: kube-proxy
 [addons] Created essential addon: kube-dns
 
 Your Kubernetes master has initialized successfully!
 
+To start using your cluster, you need to run (as a regular user):
+
+  sudo cp /etc/kubernetes/admin.conf $HOME/
+  sudo chown $(id -u):$(id -g) $HOME/admin.conf
+  export KUBECONFIG=$HOME/admin.conf
+
 You should now deploy a pod network to the cluster.
 Run "kubectl apply -f [podnetwork].yaml" with one of the options listed at:
-    http://kubernetes.io/docs/admin/addons/
+  http://kubernetes.io/docs/admin/addons/
 
-You can now join any number of machines by running the following on each node:
+You can now join any number of machines by running the following on each node
+as root:
 
-kubeadm join --token=<token> <master-ip>
+  kubeadm join --token edc833.65a928fca089fa8b 10.138.0.4:6443
```

Comment:
Let's leave `<token> <master-ip>` in here, we really don't want users copying and pasting this random token/IP.